client/src/components/PlacesAutocomplete/PlacesAutocomplete.js

```diff
@@ -18,49 +18,76 @@ type Props = {
 }
 
 // TODO: $FlowFixMe: undefined is incompatible with ...
-const PlacesAutocomplete = (props: Props) => {
+class PlacesAutocomplete extends React.Component<Props, {loaded: false}> {
```

Comment:
We prefer stateless components. If you need to do something in `componentDidMount` maybe you can use `utils.withSideEffects` wrapper?

Reply:
Yeah sure, but I guess this case is different as I also needed local state in here, while it did not make much sense to store this in global. Anyway as commented below, this probably already is fixed and we can close this PR.